e2e-tests/insurance/cypress/e2e/journeys/account/sign-in/sign-in-with-second-requested-sign-in-code.spec.js

```diff
@@ -71,14 +69,16 @@ context('Insurance - Account - Sign in - I want to enter the new security code s
       });
     });
 
-    it(`should redirect to ${DASHBOARD}`, () => {
+    it(`should redirect to ${ALL_SECTIONS}`, () => {
       cy.keyboardInput(enterCodePage[SECURITY_CODE].input(), validSecurityCode);
 
       submitButton().click();
 
-      const expectedUrl = `${baseUrl}${DASHBOARD}`;
+      cy.getReferenceNumber().then((referenceNumber) => {
+        const expectedUrl = `${baseUrl}${ROOT}/${referenceNumber}${ALL_SECTIONS}`;
 
-      cy.assertUrl(expectedUrl);
+        cy.assertUrl(expectedUrl);
+      });
```

Comment:
Can we create a new E2E test for:
- A signed in user creates an application
- User starts eligibility and creates a second application
- User should be redirected to the dashboard

I don't think this is covered